lib/typesafe.h

```diff
@@ -642,4 +646,8 @@ extern void typesafe_skiplist_del(struct sskip_head *head,
  */
 #include "typerb.h"
 
+#ifdef __cplusplus
```

Comment:
the `extern "C"` block should never cover `#include` statements (causes breakage otherwise if the other include file includes something like a system header, and the system header has its own special C++ pieces)

Reply:
Ack, somehow I didn't notice the include statement above. I'll fix this now.